src/CsvHelper.Excel.Specs/Parser/ExcelParserAsyncSpec.cs

```diff
@@ -108,7 +109,8 @@ public async void TheResultsAreCorrect()
         [Fact]
         public async void RowsHaveData()
         {
-            var csvConfiguration = new CsvConfiguration(CultureInfo.InvariantCulture, shouldSkipRecord: record => record.All(string.IsNullOrEmpty));
+            var csvConfiguration = new CsvConfiguration(CultureInfo.InvariantCulture);
+            csvConfiguration.ShouldSkipRecord = record => record.Record.All(string.IsNullOrEmpty);
```

Comment:
For newest version of CsvHelper, this should keep the inline style of coding. Because CsvConfiguration is now a Record (meaning it is immutable) : [see info](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record)